tests/Unit/NumericalAlgorithms/Spectral/Python/Test_ModalNodal.py

```diff
@@ -0,0 +1,78 @@
+#Distributed under the MIT License.
+#See LICENSE.txt for details.
+
+from spectre.Spectral import Mesh1D
+from spectre.Spectral import Basis, Quadrature
+from spectre.Spectral import modal_to_nodal_matrix, nodal_to_modal_matrix
+
+from numpy.polynomial import legendre, chebyshev
+import numpy as np
+import unittest
+
+
+class TestModalNodal(unittest.TestCase):
+    def test_modal_to_nodal_matrix_legendre_gauss(self):
+        for num_coll_points in range(2, 13):
+            mesh = Mesh1D(num_coll_points, Basis.Legendre, Quadrature.Gauss)
+            coefs = np.zeros(num_coll_points + 1)
+            coefs[-1] = 1.
+            coll_points = legendre.legroots(coefs)
```

Comment:
(optional) If you also bind `Spectral::collocation_points` you can just write here:

```py
coll_points = collocation_points(mesh)
```

What you have here is already the test for `Spectral::collocation_points`.